workitem/workitem_repository.go

```diff
@@ -31,6 +31,7 @@ const (
 
 // WorkItemRepository encapsulates storage & retrieval of work items
 type WorkItemRepository interface {
+	Exists(ctx context.Context, spaceID uuid.UUID, workitemID string) (bool, error)
```

Comment:
The workitem ID is unique on its own, so no need to add the `spaceID`. I understand that one can do permissions checking when **loading** a workitem from a space she or he is not authorized to access, but an existence check should not require this IMHO.